Dockerfile

```diff
@@ -41,9 +41,20 @@ ENV PATH ${PATH}:/home/.gimme/versions/go1.17.linux.arm64/bin:/home/.gimme/versi
 RUN go version
 
 WORKDIR /tmp/fluent-bit-$FLB_VERSION/
-RUN git clone https://github.com/matthewfala/fluent-bit.git /tmp/fluent-bit-$FLB_VERSION/
+RUN git clone https://github.com/fluent/fluent-bit.git /tmp/fluent-bit-$FLB_VERSION/
 WORKDIR /tmp/fluent-bit-$FLB_VERSION/build/
-RUN git fetch origin && git checkout 1_8_9_patch && git status
+RUN git fetch --all --tags && git checkout tags/v${FLB_VERSION} -b v${FLB_VERSION} && git describe --tags
+
+RUN git config --global user.email "falamatt@amazon.com" \
+  && git config --global user.name "Matthew Fala"
+
+# Apply Fluent Bit patches to base version
+COPY AWS_FLB_CHERRY_PICKS \
+  /AWS_FLB_CHERRY_PICKS
+
+RUN cat /AWS_FLB_CHERRY_PICKS | \
+  xargs -l bash -c 'git fetch $0 $1 && git cherry-pick $2'
```

Comment:
This cherry picks commits enumerated in added AWS_FLB_CHERRY_PICKS file.

Reply:
Can we just use the old strategy of having a specific branch? It makes it easy to check that we did everything correctly and released exactly the right code- it's simpler to code review.

Reply:
Yea, I also prefer to cut a release from a single branch.

Reply:
Discussed with Matt, I am okay with using this approach after the addition of the `git log` statement below which will print the commits that were added to the final branch that is built

Reply:
In this way, we should remember to check the `git log` result in our pipeline console even if _build_ stage succeeds.

Reply:
Here's what to look out for:
 ```
 => => # Cherry Pick Patch Summary:                                                                                                                                                                    
 => => # Commit 1 -- 88571668 tls: Issue #4098 Fix error handling for OpenSSL apis                                                                                                                     
 => => # Commit 2 -- dacbff5d downgrade mbedtls to 2.24.0                                                                                                                                              
 => => # Commit 3 -- cdccb530 io: revised support timeout on synchronous calls                                                                                                                         
 => => # Commit 4 -- 028536b8 io: default to old net_io_read                                                                                                                                           
 => => # Commit 5 -- 60098b3d aws: revised imds fallback to v1 if token request fails
```

We need to make sure that these commits match the commits listed here:
https://github.com/aws/aws-for-fluent-bit/blob/8a1989e7d20618609f213b9528d154b842b44352/AWS_FLB_CHERRY_PICKS